git-tree.py

```diff
@@ -99,15 +101,48 @@ def _parse_remote_info(self):
         except Exception as inst:
             print("An error occurred with running or parsing git show", inst)
 
+    def _parse_pr_info(self, github_pr_info):
+        if self.name not in github_pr_info:
+            self.pr_number = ""
+            self.pr_url = ""
+            self.pr_state = ""
+            return
+        self.pr_number = github_pr_info[self.name]["number"]
+        self.pr_url = github_pr_info[self.name]["url"]
+        self.pr_state = colorize_github_pr_status(github_pr_info[self.name]["state"], github_pr_info[self.name]["reviewDecision"])
+
+def github_pr_query():
+    gh_pr_list = subprocess.check_output(["gh", "pr", "list", "--state", "all", "--json", "headRefName,number,url,state,reviewDecision"])
```

Comment:
This could be a little less repetitive if you define
```
FIELDS = ["headRefName", ...]
```
then below, use as
```
for pr in gh_pr_list:
    pr_info[pr[FIELDS[0]] = {field: pr[field] for field in FIELDS[1:]}
```